notes/release-train-issue-template.md

```diff
@@ -38,6 +38,7 @@ Wind down PR queue. There has to be enough time after the last (non-trivial) PR
 ### Cutting the release
 
 - [ ] Make sure there are no stray staging repos on sonatype
+- [ ] Wait until [master build finished](https://travis-ci.org/akka/akka-http/builds/) after merging the release notes
```

Comment:
Probably it's enough to wait for it to be started and has fetched the tags from git.